### ISSUE TITLE
Feature request: JLL exposing these rootfs images

### ISSUE THREAD

**Issue by maleadt** [JuliaCI/rootfs-images#216]
I want to make PkgEval use the `package_linux` images that are built here (to eliminate differences between PkgEval and Base CI), and for that it would be convenient if there were a JLL that exposes the latest version of the available images, with automatic platform selection and everything. I'm currently just hard-linking to a specific file from the releases.

I imagine this might also be useful for Sandbox.jl, which similarly hard-links to a single rootfs now: https://github.com/staticfloat/Sandbox.jl/blob/1770a67566e61590b6c8b907430f826ff077b19c/Artifacts.toml#L17-L23

**Comment by DilumAluthge** [comment]
Maybe related to: https://github.com/staticfloat/sandbox-buildkite-plugin/issues/17

**Comment by DilumAluthge** [comment]
Ah this is different from what https://github.com/staticfloat/sandbox-buildkite-plugin/issues/17 is asking.

**Comment by DilumAluthge** [comment]
@maleadt Is this still something that you would find useful?

@staticfloat How difficult would this be to implement? Would the effort/difficulty of implementing this be worth the benefits?

**Comment by maleadt** [comment]
> Is this still something that you would find useful?

Yes. We still hard-code artifacts for the rootfs images: https://github.com/JuliaCI/PkgEval.jl/blob/e22f91fc4cab85e21d4781776b375d97a44d7198/Artifacts.toml#L17-L23, and Sandbox.jl does too.
